class URLError(Exception):
    pass


class HTTPError(URLError):
    pass
